shared/fs/folder/offline.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+import * as Kb from '../../common-adapters'
+import React from 'react'
+import * as Styles from '../../styles/index'
+
+// TODO: what is the correct thing to do when don't have any props
+class OfflineFolder extends React.PureComponent<any> {
+  render() {
```

Comment:
You can have an empty `Props` like this: `type Props = {||}`

Also, this doesn't have state or any other stuff that we need from a class component, so ideally it should be a function component:

```
const OfflineFolder = (props: Props) => (
  <Kb.Box2 ...
)
```